src/transformers/trainer.py

```diff
@@ -434,11 +434,12 @@ def _remove_unused_columns(self, dataset: "datasets.Dataset", description: Optio
             self._signature_columns += ["label", "label_ids"]
         columns = [k for k in self._signature_columns if k in dataset.column_names]
         ignored_columns = list(set(dataset.column_names) - set(self._signature_columns))
-        dset_description = "" if description is None else f"in the {description} set "
-        logger.info(
-            f"The following columns {dset_description}don't have a corresponding argument in "
-            f"`{self.model.__class__.__name__}.forward` and have been ignored: {', '.join(ignored_columns)}."
-        )
+        if len(ignored_columns):
```

Comment:
```suggestion
        if len(ignored_columns) > 0:
```
Nit, we usually explicitly test (like for all the `is not None`